extensions/medical/handleDamage.cpp

```diff
@@ -70,7 +70,7 @@ namespace ace {
                         if (amountOfDamage >= possibleInjury->minDamage && (amountOfDamage <= possibleInjury->maxDamage || possibleInjury->maxDamage <= 0))
                         {
                             if (InjuryAllowedOnSelection(possibleInjury->allowedSelections, selectionName)) {
-                                if (highestSpot == NULL) {
+                                if (!highestSpot) {
```

Comment:
please switch it to `nullptr` so that the conditional has a basic boolean type

Reply:
shared_ptr already has a boolean operator

Reply:
The more explicit a code is the better...